ansible/roles/dashd/templates/dash.conf.j2

```diff
@@ -14,6 +14,8 @@ maxconnections=256
 debug={{ dashd_debug }}
 printtoconsole=1
 
+llmq-qvvec-sync=llmq_100_67
```

Comment:
This one is to enable quorum public key shares.. Let's extract it to separate PR and we need to update MN boostrap too.